packages/server/modules/serverinvites/services/projectInviteManagement.ts

```diff
@@ -87,6 +94,15 @@ export const createProjectInviteFactory =
       throw new InviteCreateValidationError('Invalid project ID specified')
     }
 
+    // If workspace project, ensure secondaryResourceRoles are set (channeling users
+    // to the correct gql resolver)
+    const project = await deps.getStream({ streamId: resourceId })
+    if (!allowWorkspacedProjects && project && project?.workspaceId) {
```

Comment:
This prevents workspace project invite creation from the default mutation resolver